jobs/build-snaps/build-release-cdk-addons.groovy

```diff
@@ -7,7 +7,7 @@ def kube_ersion = null
 if (kube_version != "") {
     kube_ersion = kube_version.substring(1)
```

Comment:
`kube_ersion`?  Where's the `v`

Reply:
can we fix this in a drive by PR?

Reply:
It's by design:

```
kube_version=v1.23.0
kube_ersion=1.23.0
```

I will be sad the day kube_ersion goes away because it amuses me greatly. But do as you will.

Reply:
OH MY. Please leave it because it's SO clear

Reply:
LGTM